pincer/objects/guild/channel.py

```diff
@@ -177,7 +177,7 @@ class Channel(APIObject):
 
     @classmethod
     async def from_id(cls, client: Client, id: int) -> Channel:
-        data = (await client.http.get(f"/guilds/{id}")) or {}
+        data = (await client.http.get(f"/channels/{id}")) or {}

```

Comment:
Can believe that has stayed for that long!

Reply:
:trollface: